Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/submission-box/SubmissionBox.tsx

```diff
@@ -115,7 +115,13 @@ const SubmissionBox = () => {
 
     const handleOnSubmit = useCallback(async () => {
         await submit();
-    }, [ submit ]);
+        const { id: problemId } = currentProblem || {};
+        if (isNil(problemId)) {
+            return;
+        }
+
+        delete problemSubmissionCode[problemId];
```

Comment:
Mutating state like this is not acceptable in react

Reply:
Fixed.